WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -412,22 +405,52 @@ sacred_reindeer = { #Shaman
 	
 }
 
-#Franciscans
-
 wandering_hermit = {
 
 	potential = {
-		religion = fraticelli
+		OR = {
+			religion = fraticelli
+			religion = insular # Monasticism and radical asceticism were core parts of this denomination
+			religion = pelagianism
+			religion_group = indian_group # Jain nudists, Buddhist monks, etc. etc.
+			has_character_modifier = jain_naked_ascetic
+			has_character_modifier = hindu_ascetic
+			has_character_modifier = buddhist_ascetic
+			is_ascetic_trigger = yes
+		}
+		NOT = {
+			trait = cynical
+			trait = seducer
+			trait = seductress
+			trait = lustful
+		}
 	}
 
-	learning = 1
-	monthly_character_piety = 0.75
+	opposites = {
+		cynical
+		seducer
+		seductress
+		lustful	
+	}
+
+	church_opinion = 25 # Clergymen who take the extra step to travel the world for their religion are especially admired by other clergy
+
+	cannot_marry = yes
+	cannot_inherit = yes
+	lifestyle = yes
+
+	priest = yes
+	learning = 2
+	diplomacy = 2
+	monthly_character_piety = 1
 
 	customizer = no
-	random = no
+	random = yes
```

Comment:
It represents Wandering Hermits coming to join you and wander around your land. It only applies to certain characters and courts. It is not "dangerous".